src/main/java/org/opentripplanner/util/time/TimeUtils.java

```diff
@@ -115,6 +116,10 @@ public static String timeToStrCompact(int time) {
     return RelativeTime.ofSeconds(time).toCompactStr();
   }
 
+  public static String timeToStrCompact(Duration time) {
```

Comment:
Maybe rename this to `durationToStrCompact` and the param to `duration`.

Reply:
Done.